core/neat/mixins/_grid-column.scss

```diff
@@ -0,0 +1,38 @@
+/// Creates Neat a grid column of requested size.
+///
+/// @argument {number} $width
+///
+/// @argument {map} $grid
+///   The type of grid for this column. By default, the global
```

Comment:
The part of this should go away in favor of using SassDoc’s official annotation for default values:

```diff
- By default, the global `$neat-default-grid` will be used.
```